dcos/subcommand.py

```diff
@@ -219,8 +247,30 @@ def _write_package_json(pkg):
         json.dump(package_json, package_file)
 
 
+def _get_cli_binary(resources):
+    """Find compatible cli binary, if one exists
+
+    :param resources: resource.json
+    :type resources: {}
+    :returns: cli binary
+    :rtype: str | None
+    """
+    cli = resources.get("cli")
+    if cli and cli.get("binaries"):
+        arch = platform.architecture()[0]
+        if arch != "64bit":
+            logger.debug("Unsupported archecture {}\n", arch)
+            return None
+        system = platform.system().lower()
+        binary = cli["binaries"].get(system)
+        if binary:
+            return binary.get("x86-64")
+
+    return None
+
+
 def _install_env(pkg, options):
-    """ Install subcommand virtual env.
+    """Install subcommand virtual env.
```

Comment:
This comment doesn't look correct. This installs both binary and virtualenv, correct?